src/components/UI/Display/Apps/AppList/AppGrid.tsx

```diff
@@ -98,7 +98,7 @@ const AppGrid: React.FC<IAppGridProps> = ({
   }, []);
 
   const columnCount = Math.floor(width / (itemMinWidth + gridGap)) || 1;
-  const rowCount = items.length / columnCount;
+  const rowCount = Math.ceil(items.length / columnCount);
```

Comment:
This should be rounded up so the last row is displayed even if it is not filled. I tried to fix this yesterday, but made the wrong decision